### PR TITLE
update min form fields

### PR DESCRIPTION
**Pull request by CoolGoose**
the documentation creates the model in the previous pages with a required draft option

### CODE REVIEW

File: Resources/doc/getting_started/the_form_view.rst [hunk 1]
```diff
@@ -80,6 +80,7 @@ them straight away:
         $formMapper
             ->add('title', 'text')
             ->add('body', 'textarea')
+            ->add('draft', 'checkbox')
```
Comment (OskarStark) [inline Resources/doc/getting_started/the_form_view.rst]: :+1:, but would change to ``boolean``

**Comment by greg0ire** [comment]
Please rebase on the latest commit, so that our PR is green. Also consider changing "update min form fields" in your commit message to something more precise / explicit, like for instance "add missing form field in doc example"

**Comment by greg0ire** [comment]
ping @CoolGoose

**Comment by CoolGoose** [comment]
@greg0ire rebasing tomorrow, got caught with a bunch of random stuff, sorry for the delay.

**Comment by greg0ire** [comment]
> @greg0ire rebasing tomorrow, got caught with a bunch of random stuff, sorry for the delay.

No problem, take all the time you want. Just making sure you're not forgetting us :wink:

**Pull request closed without merging**